src/Config.php

```diff
@@ -17,7 +17,9 @@
 class Config
 {
     protected Paths $paths;
+
```

Comment:
Hm, I would prefer to have properties without empty lines between them. Probably you need to set `'property' => 'one'` in your fixer.